Documentation/Examples/description.md

```diff
@@ -0,0 +1,21 @@
+## Rooks Examples 
+
+
+
+
+- useWorker
```

Comment:
What is the purpose of this document 🤔 ?

Reply:
These are all the rooks that can have a better description